pkg/agent/agent.go

```diff
@@ -274,6 +292,47 @@ func (i *Initializer) validateSupportedDPFeatures() error {
 	return nil
 }
 
+func (s *SecondaryBridgeInitializer) InitializeInterfaceStore() error {
```

Comment:
maybe we could do this initialization in `pkg/agent/secondarynetwork`, and avoid the extra hook in `cmd/antrea-agent`?

Reply:
Agree with @antoninbas , it is more reasonable to move the dedicated initialization to the feature itself.